server/src/main.rs

```diff
@@ -0,0 +1,46 @@
+// WIP: not ready for reviews
+
+pub mod service;
```

Comment:
As mentioned before, maybe this should be renamed to daemon?

Reply:
Done, thanks!